plugin/resurrect/pane_tree.lua

```diff
@@ -89,6 +98,9 @@ local function insert_panes(root, panes)
 	end
 
 	root.cwd = root.pane:get_current_working_dir().file_path
+	if is_windows() then
+		root.cwd = root.cwd:gsub("^/([a-zA-Z]):", "%1:")
```

Comment:
Should not be added?

Reply:
I'm experimenting with a local branch on Windows.

So far, I've found that I needed to update the function in `init.lua`:
```lua
---@param file_name string
---@param type string
---@param opt_name string?
---@return string
local function get_file_path(file_name, type, opt_name)
	if opt_name then
		file_name = opt_name
	end
	return string.format("%s%s" .. separator .. "%s.json", pub.save_state_dir, type, file_name:gsub(separator, "+"))
end
```

I'm finding that the root.cwd is the incorrect directory on Windows and is always providing the home directory beginning with a forward slash. I'm investigating whether this is a Wezterm issue or a plugin issue at the moment.